client/flutter/lib/pages/settings_page.dart

```diff
@@ -127,6 +133,45 @@ class _SettingsPageState extends State<SettingsPage>
                         .homePagePageSliverListSettingsNotificationsInfo,
                     isToggled: _notificationsEnabled ?? false,
                     onToggle: _toggleNotifications),
+                SizedBox(
+                  height: 20,
+                ),
+                divider,
+                menuItem(
```

Comment:
Not a blocker for this PR, but in the model of the UI of the page, it feels a little weird to have individual menu items as children of the main column of the page as opposed to a menu container. It's not worth making this a fully separate widget file, but it might be worth nesting those items. But, it still does work the way it is, so this is not necessary to merge this.

Reply:
Good point and fixed